packages/integrations/netlify/readme.md

```diff
@@ -0,0 +1,27 @@
+# @astrojs/netlify
+
+Deploy your server-side rendered (SSR) Astro app to [Netlify](https://www.netlify.com/).
+
+Use this adapter in your Astro configuration file:
+
+```js
+import { defineConfig } from 'astro/config';
+import { netlifyFunctions } from '@astrojs/netlify';
+
+export default defineConfig({
+	adapter: netlifyFunctions()
+});
+```
+
+After you build your site the `dist/` folder will contain [Netlify Functions](https://docs.netlify.com/functions/overview/) in the `dist/functions/` folder. Update your `netlify.toml`:
+
+```toml
+[functions]
+  directory = "dist/functions"
+```
```

Comment:
Can we default this to `./netlify/functions` and avoid the extra configuration step? 

https://docs.netlify.com/functions/configure-and-deploy/#configure-the-functions-folder

Reply:
The reason I didn't do it that way was because it would mean changing the `config.dist`. Which is ok, but then you probably want a way to tell the adapter to *not* do that, so adding a configuration. It felt welt to have netlify have its own `dist` config, but I guess that isn't too bad.

Reply:
Ok, I made this configurable but default to changing the dist to `netlify/`.